backend/oasst_backend/prompt_repository.py

```diff
@@ -546,6 +547,8 @@ def store_text_labels(self, text_labels: protocol_schema.TextLabels) -> tuple[Te
                         message_id, protocol_schema.EmojiOp.add, protocol_schema.EmojiCode.red_flag
                     )
 
+                    discord.send_new_report_message(message=message, label_text=text_labels.text)
```

Comment:
send message_id, this would be easier for the mod team to search the DB

Reply:
I already send the message url. Is that ok?

Reply:
Yes. we could enhance it later with another PR.